invenio_app_ils/records/views.py

```diff
@@ -26,7 +26,7 @@
 class UserInfoResource(UserInfoView):
     """Retrieve current user's information."""
 
-    def response(self, user):
+    def success_response(self, user):
```

Comment:
this is a fix to be able to read the user object correctly

Reply:
Do you mind to elaborate a lit bit more?